config.ini

```diff
@@ -0,0 +1,2 @@
+[API]
```

Comment:
why you need this file? Don't we want to move to yml?

Reply:
Sorry, this was by mistake. Since we are going to use YAML config file I've removed this from the source.